pkg/util/worker_pool.go

```diff
@@ -21,6 +21,9 @@ type WorkerPool interface {
 	// functions have returned so synchronization must be achieved
 	// via other means (e.g. sync.WaitGroup).
 	Release()
+
+	// Tune changes the capacity of this pool.
+	Tune(int)
```

Comment:
does this change solve something?

Reply:
https://github.com/nspcc-dev/neofs-node/blob/b13fa5af7eb07fc94909e7e3b8ea9331114956d7/pkg/local_object_storage/engine/control.go#L204-L206
`e.shardPools` are `WorkerPools`, and I need to `Tune`.